src/base/bittorrent/torrentimpl.h

```diff
@@ -308,7 +309,7 @@ namespace BitTorrent
         // we will rely on this workaround to remove empty leftover folders
         QHash<lt::file_index_t, QVector<QString>> m_oldPath;
 
-        QHash<QString, TrackerInfo> m_trackerInfos;
+        QHash<QString, QMap<lt::tcp::endpoint, int>> m_trackerPeerCounts;
```

Comment:
Why not use `QHash`? You don't need to maintain order and `lt::tcp::endpoint` seems small enough that both QHash and QMap would perform quite similar in performance.

Reply:
>Why not use `QHash`?

I just don't want to maintain `qHash()` for `tcp::endpoint`.
>`lt::tcp::endpoint` seems small enough that both QHash and QMap would perform quite similar in performance.

Yes. So I would keep use QMap to keep things easier.